File: buff/classes/354/pub/sum/csharp/Sum.cs
// C# sum program

public class Sum {

    public static int sum(int[] seq) {
	int s=0;
	foreach (int v in seq)
	    s+=v;
	return s;
    }

    public static void Main() {
	int[] seq=new int[] {5,6,1,8,3,7};
	System.Console.WriteLine(sum(seq));
    }

}

File: buff/classes/354/pub/sum/rust/sum.rs
// Rust sum program

fn sum(seq: &[i32]) -> i32 {
  let mut s=0;
  for v in seq.iter() {
    s+=v;
  }
  s
}

fn main() {
  let seq=[5,6,1,8,3,7];
  println!("{}",sum(&seq));
}
